tests/components/ituran/test_sensor.py

```diff
@@ -0,0 +1,76 @@
+"""Test the Ituran device_tracker."""
+
+from unittest.mock import AsyncMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+from pyituran.exceptions import IturanApiError
+import pytest
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.components.ituran.const import UPDATE_INTERVAL
+from homeassistant.const import STATE_UNAVAILABLE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed, snapshot_platform
+
+
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_sensor(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    mock_ituran: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test state of sensor."""
+    with patch("homeassistant.components.ituran.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
+    await snapshot_platform(hass, entity_registry, snapshot, mock_config_entry.entry_id)
+
+
+@pytest.mark.parametrize(
+    ("entity_id"),
+    [
+        "sensor.mock_model_address",
+        "sensor.mock_model_battery_voltage",
+        "sensor.mock_model_heading",
+        "sensor.mock_model_last_update",
+        "sensor.mock_model_mileage",
+        "sensor.mock_model_speed",
+    ],
+)
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_availability(
```

Comment:
I think I'd prefer looping over parametrization here (as it's a bit quicker and they all share the same availability). Like loop over all sensors to check availability -> make the error and wait -> loop again over all of them

Reply:
Done